libs/langchain-scripts/src/build_v2.ts

```diff
@@ -441,16 +441,16 @@ async function checkTreeShaking(config: LangChainConfig) {
   );
   const externals = listExternals(packageJson, config?.internals ?? []);
   const entrypoints = listEntrypoints(packageJson);
-  const consoleLog = console.log;
-  /** @type {Map<string, { log: string; hasSideEffects: boolean; }>} */
+  const consoleInfo = console.info;
+  /** @type {Map<string, { log: string; hasUnexpectedSideEffects: boolean; }>} */
   const reportMap = new Map();
```

Comment:
This can be replaced w/ an actual generic passed to Map. Any jsdocs like this in this file are leftover from the days when this was a `.js` file
```suggestion
  const reportMap = new Map<string, { log: string; hasUnexpectedSideEffects: boolean; }>();
```

Reply:
Not worth it